src/utils/database.ts

```diff
@@ -192,7 +192,6 @@ interface TranscriptComponent {
 
 interface TranscriptAuthor {
     username: string;
-    discriminator: number;
```

Comment:
See other discriminator removal comment, it may be better to make this optional than remove it